conduit/src/main/java/io/rtr/conduit/amqp/impl/AMQPConsumerBuilder.java

```diff
@@ -239,6 +245,8 @@ protected boolean isPoisonQueueEnabled() {
         return poisonQueueEnabled;
     }
 
+    protected boolean getExclusive() { return exclusive; }
```

Comment:
nit: the getters for everything else look to be located immediately below their setter